aws/cf_template.go

```diff
@@ -447,14 +447,17 @@ func newTargetGroup(spec *stackSpec, targetPortParameter string) *cloudformation
 	protocol := "HTTP"
 	healthCheckProtocol := "HTTP"
 	healthyThresholdCount, unhealthyThresholdCount := spec.albHealthyThresholdCount, spec.albUnhealthyThresholdCount
+	protocolVersion := cloudformation.String("HTTP1")
```

Comment:
Maybe we should keep it `nil` to be backward compatible?

Reply:
Could be problematic, consider the case going back from H2 to H1 where CF might not detect that config change due to the parameter being nil. I think the default value will not become mandatory again!?